pkg/csi/service/vanilla/controller.go

```diff
@@ -284,6 +297,9 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		return err
 	}
 
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.ListViewPerf) {
+		go cnsvolume.ClearInvalidTasksFromListView(multivCenterCSITopologyEnabled)
```

Comment:
I don't see this implementation

Reply:
Please see the implementation here -> https://github.com/adikul30/vsphere-csi-driver/blob/reduce-vpxd-connections/pkg/common/cns-lib/volume/manager.go#L284-L299